src/Conversion/ONNXToKrnl/Tensor/Transpose.cpp

```diff
@@ -33,7 +33,8 @@ struct ONNXTransposeOpLowering : public ConversionPattern {
 
     // Get a shape helper.
     ONNXTransposeOpShapeHelper shapeHelper(&transposeOp, &rewriter);
-    assert(succeeded(shapeHelper.Compute(operandAdaptor)));
+    auto shapecomputed = shapeHelper.Compute(operandAdaptor);
```

Comment:
nit - we might want to add ```(void)shapecomputed;``` to avoid compiler warnings such as "unused var" in release builds. It's a quite commonly-used LLVM coding convention, e.g.

https://llvm.org/docs/CodingStandards.html#assert-liberally